modular_skyrat/modules/goofsec/code/sec_clothing_overrides.dm

```diff
@@ -63,6 +63,15 @@
 			RESKIN_WORN_ICON_STATE = "backpack_security_white",
 			RESKIN_INHAND_STATE = "backpack_security_white"
 		),
+		//BUBBER EDIT ADDITION - RED SEC
+		"Red Variant" = list(
+			RESKIN_ICON = 'icons/obj/storage/backpack.dmi',
+			RESKIN_ICON_STATE = "backpack-security",
+			RESKIN_WORN_ICON = 'icons/mob/clothing/back/backpack.dmi',
+			RESKIN_INHAND_L = 'icons/mob/inhands/equipment/backpack_lefthand.dmi',
+			RESKIN_INHAND_R = 'icons/mob/inhands/equipment/backpack_righthand.dmi',
+			RESKIN_INHAND_STATE = "securitypack"
+		),
```

Comment:
Gonna use this snippet as an example. Going from the top of my head so this might not be exact but either way gives a good example of how to do modularization of lists

```suggestion
/obj/item/storage/backpack/security/Initialize(mapload)
	. = ..()
	unique_reskin["Red Variant"] = list(
			RESKIN_ICON = 'icons/obj/storage/backpack.dmi',
			RESKIN_ICON_STATE = "backpack-security",
			RESKIN_WORN_ICON = 'icons/mob/clothing/back/backpack.dmi',
			RESKIN_INHAND_L = 'icons/mob/inhands/equipment/backpack_lefthand.dmi',
			RESKIN_INHAND_R = 'icons/mob/inhands/equipment/backpack_righthand.dmi',
			RESKIN_INHAND_STATE = "securitypack"
		),
```

Reply:
(Note: If this doesn't work it's probably due to some pre-initialization of the reskins, in which case I'll look into it proper)

Reply:
Looks like it's working fine! I now see what I did wrong with my original Modu attempt. Thanks, chief. Will get this updated soon.